Remove torch.distributed.tensor.OpSchema.has_symints

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack/tree/0.12.0) (oldest at bottom):
* __->__ #163667
* #162990

It appears to be unused based on `cd torch; rg has_symints`.

cc @H-Huang @awgu @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @pragupta @ezyang @msaroufim @dcci

swolchok committed:
Remove torch.distributed.tensor.OpSchema.has_symints

It appears to be unused based on `cd torch; rg has_symints`.

[ghstack-poisoned]

xmfan reviewed (approved):
lgtm if ci passes

azahed98 reviewed (approved):
lgtm, apologies for missing these!

swolchok committed:
fix missing torch.distributed.fsdp import in test that used torch.ops.fsdp on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
Update on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
rebase on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
Update on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
rebase to pick up reapply of dependency on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
Update on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
Update on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
fix bottom-of-stack PR on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
fix dependency on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
Update on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
partially fix pickling, but tests are still broken on "Remove torch.distributed.tensor.OpSchema.has_symints"


It appears to be unused based on `cd torch; rg has_symints`.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k pragupta ezyang msaroufim dcci

[ghstack-poisoned]

swolchok committed:
Update

[ghstack-poisoned]
